scripts/teardown-workloads.sh

```diff
@@ -2,7 +2,20 @@
 
 set -euo pipefail
 
-kubectl delete -f tests/workloads-common --wait --timeout 60s --ignore-not-found=true
-kubectl delete -f tests/workloads-pushed-using-docker-build-push --wait --timeout 60s --ignore-not-found=true
-kubectl delete -f tests/workloads-pushed-using-spin-registry-push --wait --timeout 60s --ignore-not-found=true
+case "$1" in
```

Comment:
This highlights that it would be nice to have all the tests in one directory regardless of the K8s runtime rather than duplicating